Lombiq.Tests.UI/Docs/FakeVideoCaptureSource.md

```diff
@@ -15,15 +15,17 @@ _Hint: The `mjpeg` format will usually result in a smaller file size._
 ffmpeg -y -i test.mp4 -pix_fmt yuv420p test.y4m
 
 # Convert with resize to 480p.
-ffmpeg -y -i test.mp4 -filter:v scale=480:-1 -pix_fmt yuv420p test.y4m
+ffmpeg -y -i test.mp4 -vf "scale=480:720" -pix_fmt yuv420p test.y4m
 
 # Convert mp4 to mjpeg.
 ffmpeg -y -i test.mp4 test.mjpeg
 
 # Convert with resize to 480p.
-ffmpeg -y -i test.mp4 -filter:v scale=480:-1 test.mjpeg
+ffmpeg -y -i test.mp4 -vf "scale=480:720" test.mjpeg
 ```
 
+_Warning: Using the `-filter:v scale=480:-1` command might "ruin" the video, resulting in a black screen in the browser without warnings._
```

Comment:
```suggestion
> ⚠ Using the `-filter:v scale=480:-1` command might "ruin" the video, resulting in a black screen in the browser without warnings.
```

We've used these textbook style message boxes (quote block starting with ⚠ for warning or ℹ️ for information) in other projects including Hastlayer. I think it's more eye catching without being tacky.
Also you could replace the `_Hint: ` above with `> ℹ️`. (tbh I hate writing "hint:" because I think it can be read as condescending)

Reply:
I agree.